### PR TITLE
ONI-114:Basic setup.

### PR DESCRIPTION
**Pull request by ghost**
Ticket: https://openimis.atlassian.net/browse/ONI-116

Changes: 
- Basic language pack structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # openimis-fe-nepal_language_pack_js
 openIMIS translation to Nepali and english version of Nepal implementation.
```
Comment (dragos-dobre) [inline README.md]: English with big E.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+
+
+INSERT INTO tblLanguages ("LanguageCode", "LanguageName", "SortOrder") VALUES ('np', 'Nepali', 3);
```
Comment (dragos-dobre) [inline README.md:8]: Why this here?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:8]: I've updated the repository, you have to insert language information manually, so I left the code for convenience.

Comment (dragos-dobre) [reply README.md:8]: @purusap can you confirm that you didn't had the language in tblLanguages?

Comment (ghost) [reply README.md:8]: This step is mentioned in the link with instructions in the assembly module:
`in tblLanguages on database level add new language for example 'es' (Spanish language)`
So we can assume someone will need it.

Comment (dragos-dobre) [reply README.md:8]: Once the mobile apps are migrated to modular FHIR, I suppose we won't need tblLanguages anymore. Will check with BS about this.

**Comment by ghost** [comment]
@delcroip @dragos-dobre 
I've recreated this PR so that branch develop is created:
https://github.com/openimis/openimis-fe-nepal_language_pack_js/pull/2

File: README.md (hunk 2 above)
Comment (purusap) [reply README.md:8]: > @purusap can you confirm that you didn't had the language in tblLanguages?

We do have "ne" Langauge in tblLanguages.

**Pull request merged**